qiita_db/processing_job.py

```diff
@@ -186,13 +253,13 @@ def status(self):
             qdb.sql_connection.TRN.add(sql, [self.id])
             return qdb.sql_connection.TRN.execute_fetchlast()
 
-    @status.setter
-    def status(self, value):
+    def _set_status(self, value):
         """Sets the status of the job
 
         Parameters
         ----------
-        value : str, {'queued', 'running', 'success', 'error'}
+        value : str, {'queued', 'running', 'success', 'error',
```

Comment:
Should we check that the value of `value` is valid? Seems like this could have caught the typo that's being fixed in `test_processing_job.py` unless that's totally unrelated.